### ISSUE TITLE
Maybe it should be updated.

### ISSUE THREAD

**Issue by LvAnchorage** [forza-painter/forza-painter#19]
In the latest version of the game, this program is not working, which I think is related to the recent update
![微信图片_20231206204157](https://github.com/forza-painter/forza-painter/assets/153088830/981028f8-5b0d-4106-bcac-1ec8fa6083c7)

**Comment by DUCKDK1** [comment]
yeah man,please update it.EVERYDAY without using it to paste my favorite anime figure on my favorite car,man the only thing ican do is in the toilet playing with myself and then into sage time you know???
